mantidimaging/gui/windows/spectrum_viewer/view.py

```diff
@@ -108,6 +110,9 @@ def __init__(self, main_window: MainWindowView):
         self.normaliseCheckBox.stateChanged.connect(self.normaliseStackSelector.setEnabled)
         self.normaliseCheckBox.stateChanged.connect(self.presenter.handle_enable_normalised)
         self.normaliseCheckBox.stateChanged.connect(self.presenter.handle_button_enabled)
+        self.normalise_ShutterCount_CheckBox.stateChanged.connect(self.normalise_ShutterCount_CheckBox.setDisabled)
```

Comment:
Is this still needed? Looks like `handle_button_enabled` calls `self.normalise_ShutterCount_CheckBox.setEnabled` and will overwrite it.

Reply:
you're right, this is no longer needed